server/index.js

```diff
@@ -14,30 +16,69 @@ app.use(express.json());
 
 app.use('/items/:item_id', express.static('react-client/dist'));
 
-app.get('/item/images', (req, res)=>{
-  pool.query('SELECT * FROM image_urls', (err, data) => {
-    if (err){
+
+
+app.get('/item/:item_id/images', (req, res)=> {
+  const id = req.params.item_id;
+  controller.images.getOne(id)
+    .then((response) => {
+      console.log(response)
+      res.send(response);
+    })
+    .catch((err) => {
       res.status(400).send(err);
-    } else {
-      res.status(200).send(data)
-    }
-  });
+    });
 });
 
+app.post('/item/:item_id/images', (req, res)=>{
+  const id = req.body.item_id;
+  const url = req.body.url;
+  controller.images.post(id, url)
+    .then((response) => {
+      console.log(response.data)
+      res.send(response);
+    })
+    .catch((err) => {
+      res.status(400).send(err);
+    });
+});
 
-app.get('/item/:item_id/images', (req, res)=>{
-  const values = [`${req.params.item_id}`];
-  pool.query(`SELECT * FROM image_urls WHERE item_id = ${req.params.item_id}`, (err,data) => {
+app.put('/item/:item_id/images', (req, res)=>{
+  //const id = req.params.item_id;
+  controller.images.update(index, url)
+    .then((response) => {
+      console.log(response.data)
+      res.send(response);
+    })
+    .catch((err) => {
+      res.status(400).send(err);
+    });
+});
+
+app.delete('/item/:item_id/images', (req, res)=>{
+  //const id = req.params.item_id;
+  controller.images.delete(index)
+    .then((response) => {
+      console.log(response.data)
+      res.send(response);
+    })
+    .catch((err) => {
+      res.status(400).send(err);
+    });
+});
+
+app.get('/item/images', (req, res)=>{
+  pool.query('SELECT * FROM images', (err, data) => {
```

Comment:
why have the query on the server when you also have a controller/model workflow?

Reply:
Agreed. These endpoints are for the other services, and I was trying to figure out why they needed them first but forgot to loop back. Will correct!